packages/@react-spectrum/list/docs/ListView.mdx

```diff
@@ -0,0 +1,478 @@
+Double clicking matches the behavior of desktop platforms like macOS and Windows, and a separate selection mode matches mobile platforms like iOS and Android.
+
+```tsx example
+// Checkbox selection with onAction
+<Flex direction="column" gap="size-300">
+  <PokemonList onAction={key => alert(`Opening item ${key}...`)} aria-label="Checkbox selection list with row actions" />
+  <PokemonList selectionStyle="highlight" onAction={key => alert(`Opening item ${key}...`)} aria-label="Highlight selection list with row actions" />
+</Flex>
+```
+
+## Drag and drop
+
+To enable drag and drop in a ListView, you must provide the respective drag and drop hooks sourced from `useDragHooks` and `useDropHooks` to the ListView's `dragHooks` and `dropHooks` props respectively.
+The example below demonstrates how to create a ListView that supports reordering its rows via drag and drop. For more information on the hooks themselves and the various supported ways to perform a drag and drop interaction,
+please see the [following page](dragAndDrop.html).
+
+```tsx example
+import {useDragHooks, useDropHooks} from '@react-spectrum/dnd';
+import {useListData} from '@react-stately/data';
+
+function DragAndDropExample() {
+  let list = useListData({
+    initialItems: [
+      {id: '1', type: 'file', name: 'Adobe Photoshop'},
+      {id: '2', type: 'file', name: 'Adobe XD'},
+      {id: '3', type: 'folder', name: 'Documents'},
+      {id: '4', type: 'file', name: 'Adobe InDesign'},
+      {id: '5', type: 'folder', name: 'Utilities'},
+      {id: '6', type: 'file', name: 'Adobe AfterEffects'}
+    ]
+  });
+
+  // Update the list order based on the position of the drop and the set of rows that were dropped
+  let onMove = (keys, target) => {
+    if (target.dropPosition === 'before') {
+      list.moveBefore(target.key, keys);
+    } else {
+      list.moveAfter(target.key, keys);
+    }
+  };
+
+  let dragHooks = useDragHooks({
+    getItems(keys) {
+      return [...keys].map(key => {
+        let item = list.getItem(key);
+        // Setup the info for each item that will be propagated upon drag
+        return {
+          'text/plain': item.name,
+          [item.type]: item.name,
+          'id': item.id
+        };
+      });
+    }
+  });
+
+  let dropHooks = useDropHooks({
+    async onDrop(e) {
+      if (e.target.type !== 'root' && e.target.dropPosition !== 'on') {
+        let keysToMove = [];
+        // Only add the dropped item to the list of keys to move
+        // if the dropped item has the expected types and fields
+        for (let item of e.items) {
+          if (item.kind === 'text' && item.types.has('text/plain')) {
+            if (item.types.has('folder') || item.types.has('file')) {
+              let key = await item.getText('id');
+              keysToMove.push(key);
+            }
+          }
+        }
```

Comment:
Open to opinions on whether or not I should adjust this logic to work around the Android Chrome drag and drop bug where it only works with `text/plain`. Is it worthwhile working around that bug in this example? Maybe I just check if types has `text/plain` and `id` as types...